File: EmployeeManagement/App/repositories/EmployeeRepository.ts
import { Employee as EmployeeModel } from 'models/Employee';
import { Repository } from './repository';
import { IRawEmployee } from '../models/raw/IRawEmployee';
import { IEmployeeRepository } from './interfaces/IEmployeeRepository';

export class EmployeeRepository extends Repository<IRawEmployee, EmployeeModel> implements IEmployeeRepository {
    protected specificUrl(): string {
        return 'employee';
    }
}

File: EmployeeManagement/Repositories/EmployeeRepository.cs
using System.Data.Entity;
using System.Linq;
using EmployeeManagement.Models;
using EmployeeManagement.Repositories.Interfaces;

namespace EmployeeManagement.Repositories
{
    public class EmployeeRepository : Repository<Employee>, IEmployeeRepository
    {
        public bool Exists(int id)
        {
            return DbSet.Count(e => e.Id == id) > 0;
        }
    }
}